src/User.php

```diff
@@ -1185,6 +1198,14 @@ public function post_updateItem($history = true)
                 true
             );
         }
+
+        // Hash user_dn if is updated
+        if (in_array('user_dn', $this->updates)) {
+            $this->update([
+                'id' => $this->fields['id'],
+                'user_dn_hash' => md5($this->fields['user_dn'])
```

Comment:
Can be updated to null, I guess.
```suggestion
                'user_dn_hash' => is_string($this->fields['user_dn']) && strlen($this->fields['user_dn']) > 0 ? md5($this->fields['user_dn']) : '',
```